README.md

```diff
@@ -27,63 +47,99 @@ You will need to perform the following on your development machine:
-| `start`             | Runs complete test and build suite and starts the dev server.                                 |
-| `serve`             | Runs build suite and starts the dev server.                                                   |
-| `build`             | Runs build suite and outputs the result into `dist`. Takes optional `base` value as argument. |
-| `test`              | Runs the test suite with eslint, and stylelint.                                               |
-| `test:eslint`       | Runs only eslint.                                                                             |
-| `test:stylelint`    | Runs only stylelint.                                                                          |
-| `test:mocha`        | Runs only the test suite.                                                                     |
-| `deploy`            | Runs `build` script, then pushes the output to GH Pages.                                      |
-| `deploy:clean`      | Runs `build` script, clears `gh-pages` branch, then pushes the output to GH Pages.            |
-| `deploy:folder`     | Runs `build` script, then pushes the output to the specified folder in GH Pages.              |
-| `deploy:staging`    | Initiates procedures for deploying to staging. (Package specific)                             |
-| `deploy:production` | Initiates procedures for deploying to production. (Package specific)                          |
-
-**Please follow the README of each package you intend to work with on how to get set up.** However, the above scripts can be run from the root directory in the following manner.
+| Package param | Command             | Description                                                                                   |
+| :-----------: | ------------------- |:---------------------------------------------------------------------------------------------:|
+| ✅            | `start`             | Runs complete test and build suite and starts the dev server.                                 |
+| ✅            | `serve`             | Runs build suite and starts the dev server.                                                   |
+| ✅            | `build`             | Runs build suite and outputs the result into `dist`. Takes optional `base` value as argument. |
+| ✅            | `test`              | Runs the test suite with eslint, and stylelint.                                               |
+| ✅            | `test:eslint`       | Runs only eslint.                                                                             |
+| ✅            | `test:stylelint`    | Runs only stylelint.                                                                          |
+| ✅            | `test:mocha`        | Runs only the test suite.                                                                     |
+| ❌            | `deploy`            | Runs `build` script, then pushes the output to GH Pages.                                      |
```

Comment:
why [X]?

Reply:
That column is about whether that script accepts a `package param` or not. And the ones with `X` do not.